src/dotnet/Program.cs

```diff
@@ -260,14 +260,12 @@ private static void PrintVersion()
 
         private static void PrintInfo()
         {
-            HelpCommand.PrintVersionHeader();
-
             DotnetVersionFile versionFile = DotnetFiles.VersionFileObject;
             var commitSha = versionFile.CommitSha ?? "N/A";
             Reporter.Output.WriteLine();
-            Reporter.Output.WriteLine("Product Information:");
-            Reporter.Output.WriteLine($" Version:            {Product.Version}");
-            Reporter.Output.WriteLine($" Commit SHA-1 hash:  {commitSha}");
+            Reporter.Output.WriteLine(".NET Core SDK (reflecting any global.json):");
```

Comment:
I kind of bought the non-localization of "key: value" pairs for parsing, but I feel like "Product Information and Runtime Environment should have been localized. I also think that the explanation "reflecting any global.json" should be localized for similar reasons.

Reply:
Same here.

Reply:
Is it worth being explicit about whether a `global.json` file was picked up and is affecting this version? Or is it okay to let the user try and figure out whether a `global.json` is affecting the SDK version.

Reply:
That's a good question I am think eventually we should try to indicate to the user that a global.json is being used. However, not easily done by the SDK itself. By the time the code reaches the SDK, the host has already done the work to figure out which SDK to pick. We don't currently have an API to figure that out. I think this is a fine first turn of the crank and your suggestion can come in on another iteration. We have been looking into having APIs for this sort of thing for other reasons as well.